packages/sysutils/busybox/sleep.d.serial/99-suspend-modules.sh

```diff
@@ -3,8 +3,17 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
 
+if [ -f /storage/.config/suspend-modules.conf ]; then
+    . /storage/.config/suspend-modules.conf
+    custom_modules="${SUSPEND_MODULES}"
```

Comment:
One suggestion, set `SUSPEND_MODULES=""` after line 8 - if in future the system list is disabled or removed, the custom list will double up.

Reply:
Good catch. Amended.